### PR TITLE
adding support for sqlite3

### PR DESCRIPTION
**Pull request by csepulv**
I've been using bookshelf-json-columns and started to use sqlite in memory DB's in my tests. Unfortunately, I ran into  some issues as I am using sqlite3 module and not sqlite, so I submit this pull request to add sqlite3

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default Bookshelf => {
     }
   });
 
-  if (client === 'sqlite') {
+  if (client === 'sqlite' || client === 'sqlite'3) {
```
Comment (ricardogama) [inline src/index.js]: Can you fix the syntax error here?

**Comment by ricardogama** [comment]
Hi @csepulv! 

We're also using **sqlite3** as dependency to test the plugin with SQLite, you can easily fix this by changing `sqlite3` to `sqlite` on your *knexfile*.

However, the plugin should accept all clients and aliases **knex** supports, so I'll be glad to merge your PR, cheers!

**Comment by csepulv** [comment]
Good to know about the sqlite change. Sorry about the typo.

**Comment by ricardogama** [comment]
@csepulv Released as [1.1.1](https://github.com/seegno/bookshelf-json-columns/releases/tag/1.1.1), cheers!

**Pull request merged**